src/api/init.ts

```diff
@@ -128,6 +137,14 @@ export async function startApiServer(opts: {
     });
     app.use(promMiddleware);
   }
+  // adding stacks-api-version in header
+  app.use((_, res, next) => {
+    if (stacksVersion) {
+      res.setHeader('X-Stacks-API-Version', stacksVersion);
+    }
+    res.setHeader('Access-Control-Expose-Headers', 'X-Stacks-API-Version');
```

Comment:
@MHassanTariq this would overwrite previous values for this header. Can you modify it so that it appends this new value?